README.md

```diff
@@ -0,0 +1,191 @@
+# Xmartlabs' Android Style Guide
+
+This is the Xmartlabs' Android and Java Styles Guide. These are conventions in general followed by the team.
+
+## Java
+
+The idea is to follow [Google Java Style](http://google.github.io/styleguide/javaguide.html), but with some changes and adding ideas.
+
+### Changes
+
+#### Vertical whitespace
+
+There must be no vertical whitespace before the first member and after the last one of a class.
+
+```java
+public class ClassExample {
+  private int firstMember; // no vertical whitespace before this line
+
+  // ...
+
+  public void lastMember() {
+    // ...
+  } // no vertical whitespace after this line
+}
+```
+
+Multiple vertical whitespaces must never be used.
+
+#### Array initializers
+
+The preferred array initializer "block-like construct" style when it doesn't fit in one line is one value per line:
+
+```java
+new int[] {
+  0,
+  1,
+  2,
+  3,
+}
+```
+
+#### Numeric literals
+
+`float`-valued literals use a lowercase `f` suffix: `100.23f`.
+
+#### Class and interface names
+
+If it's a list of an entity, use singular: `CustomerListFragment` for instance.
+
+#### Field, method, variable and parameter names
+
+If you need to name a `List` of activities for example, just name it `activities` and if it's `ListView`: `activitiesListView`. So we use plural here.
+
+#### Local variable names
+
+One-character names are only permitted if they are `i`, `j` or `k` indexes in loops. They must be used in this respective order when nesting them. Example:
```

Comment:
I think it will make the code less understandable and error pruner, the loop variable could be used much later in the code, then:
* I think defining a new variable like "boolean itemFound" should be easier to understand
* if you change the stop condition on the for, you could forget to change the `if` statement latter.